sunpy/map/header_helper.py

```diff
@@ -24,7 +27,7 @@ def meta_keywords():
          'cunit2': 'Units of the coordinate increments along naxis2 e.g. arcsec **required',
```

Comment:
I think this example can go as well. 

Or we need to catch the waring but I forgot how to do that for doctests.

Reply:
i think a `# doctest: +SKIP` should do that?

Reply:
regardless, i guess we should just remove the example since we're deprecating the function